operators/pkg/controller/elasticsearch/client/v6.go

```diff
@@ -94,6 +97,15 @@ func (c *clientV6) DeleteVotingConfigExclusions(ctx context.Context, waitForRemo
 	return errors.New("Not supported in Elasticsearch 6.x")
 }
 
+func (c *clientV6) Request(ctx context.Context, r *http.Request) (*http.Response, error) {
```

Comment:
Might be worth having a comment here explaining that request.URL should just be the path (including the `/` prefix), not the full URL, is that right? Also I'm surprised our linting doesn't get upset about exported funcs without a comment

Reply:
I guess the linter is smart enough to figure out that the docs are on the `interface`